src/components/ZclAttributeManager.vue

```diff
@@ -276,13 +279,33 @@ export default {
         return false
       }
     },
-    isDisabled(id, name, selectedClusterId, selectedCluster) {
+    isDisabledDefault(id, name, selectedClusterId) {
+      return (
```

Comment:
return (isDisabled() ||
         this.selectionStorageOption[
           this.hashAttributeIdClusterId(id, selectedClusterId)
         ] == 'External')
         
         
         avoid the duplicate code

Reply:
which part is duplicated? this case requires more conditions..

ill see what I can do to clear it up, thanks

Reply:
Isn't `!this.selection.includes(
         this.hashAttributeIdClusterId(id, selectedClusterId)
       )` part the same in `isDisabled(id, selectedClusterId)` and ` isDisabledDefault(id, name, selectedClusterId)` ?

Reply:
yes but this is a different set of conditions. 

do you think I should somehow use isDisabled within isDisabledStorage?

I think that would cause more confusion

Reply:
isDisabled has a condition

isDisabledStorage has that same condition plus another condition

isDisabledDefault has both of the conditions plus another one


I could make isDisabled do all of this work but it would be much harder to read imo

Reply:
Actually, Vue has limitations which would make it even harder to have isDisabled do all of this work. Ill look more into it though...